fibsem/milling/core.py

```diff
@@ -19,11 +19,33 @@
 
 ########################### SETUP
 
+def get_stage_reference_image(
+    microscope: FibsemMicroscope, milling_stage: FibsemMillingStage
+) -> FibsemImage:
+    ref_image = milling_stage.reference_image
+    if ref_image is None:
+        image_settings = ImageSettings(
+            hfw=milling_stage.milling.hfw,
+            dwell_time=1e-6,
+            resolution=[1536, 1024],
+            beam_type=milling_stage.milling.milling_channel,
+            reduced_area=milling_stage.alignment.rect,
+            path=fcfg.DATA_CC_PATH,  # TODO: set this to the last-path?
```

Comment:
This path can probably be updated to `milling_stage.imaging.path', if it's None then use the DATA_CC_PATH